layouts/css/page-modules/_header.scss

```diff
@@ -186,18 +203,8 @@ header {
       height: 1.5em;
       padding: 0;
     }
-  }
-}
-
-#logo {
-  display: inline-block;
-
-  &:hover {
-    background-color: transparent;
-    text-decoration: underline;
-  }
-
-  img {
-    margin: .5em auto 0;
+    .logo{
```

Comment:
Why is there other `.logo` tag here?

Reply:
I pretty much believe you can add this `justify-content` on the first `.logo` definition